src/credssp/mod.rs

```diff
@@ -502,18 +513,22 @@ impl<C: CredentialsProxy<AuthenticationData = AuthIdentity>> CredSspServer<C> {
                 let mut output_token = vec![SecurityBuffer::new(Vec::with_capacity(1024), SecurityBufferType::Token)];
 
                 let mut credentials_handle = self.credentials_handle.take();
+                let sspi_ctx = &mut self.context.as_mut().unwrap().sspi_context;
                 match try_cred_ssp_server!(
-                    self.context
-                        .as_mut()
-                        .unwrap()
-                        .sspi_context
-                        .accept_security_context()
-                        .with_credentials_handle(&mut credentials_handle)
-                        .with_context_requirements(ServerRequestFlags::empty())
-                        .with_target_data_representation(DataRepresentation::Native)
-                        .with_input(&mut [input_token])
-                        .with_output(&mut output_token)
-                        .execute(),
+                    AcceptSecurityContext::<
+                        '_,
+                        _,
+                        WithoutCredentialsHandle,
+                        WithoutContextRequirements,
+                        WithoutTargetDataRepresentation,
+                        WithoutOutput,
+                    >::new()
```

Comment:
thought: Not very ergonomic. I would expect that we don’t need to specify all the generic parameters.